deployers/deploymentreader.go

```diff
@@ -65,8 +65,17 @@ func (reader *DeploymentReader) bindPackageInputsAndAnnotations() {
 
 	if reader.DeploymentDescriptor.Application.Packages == nil {
 		// a single package is specified in deployment YAML file with "package" key
-		packMap[reader.DeploymentDescriptor.Application.Package.Packagename] = reader.DeploymentDescriptor.Application.Package
-        utils.PrintOpenWhiskOutputln("WARNING: The package YAML key in deployment file will soon be deprecated. Please use packages instead as described in specifications.")
+        if len(reader.DeploymentDescriptor.Application.Package.Packagename) != 0 {
```

Comment:
Can we better comment and align the if-else blocks?

Reply:
Done.